### PR TITLE
fix(locksmith) adding migration fix

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding missing migration, wrapped in `try/catch` in case it already exists.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/migrations/20230208144216-fix-missing-index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  },
+
+  async down(queryInterface, Sequelize) {
+    /**
+     * Add reverting commands here.
+     *
+     * Example:
+     * await queryInterface.dropTable('users');
+     */
```
Comment (searchableguy) [inline locksmith/migrations/20230208144216-fix-missing-index.js:32]: Add one for removal too. :)

**Review by searchableguy — APPROVED** [review]

**Review by kalidiagne — APPROVED** [review]

File: locksmith/migrations/20230208144216-fix-missing-index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
```
Comment (searchableguy) [inline locksmith/migrations/20230208144216-fix-missing-index.js:33]:
```suggestion
    queryInterface.removeIndex('CustomEmailContents', 'lock_network_template_index')
  }
```

Comment (julien51) [reply locksmith/migrations/20230208144216-fix-missing-index.js:33]: Hum, I wonder if this would break the other migration though?

Comment (julien51) [reply locksmith/migrations/20230208144216-fix-missing-index.js:33]: Probably not in fact.

**Pull request merged**